components/Comments/Comment.vue

```diff
@@ -118,22 +131,35 @@
       },
       isOwner: {
         type: Boolean
+      },
+      post: {
+        type: Object
+      },
+      depth: {
+        type: Number,
+        default: 0
       }
     },
     data () {
+      const styleBackground = {};
+      if (this.depth > 0) {
+        styleBackground.backgroundColor = '#f2efef'
```

Comment:
Please set styles in the style section and work with classes inside the code.

Reply:
How to set a style which depends on a js var? 
I think you cannot do this since sass is a preprocessor. So I have to do it with js..

Reply:
I moved it to the style section